src/OrchardCore.Themes/TheAdmin/Assets/js/TheAdmin/TheAdmin.ts

```diff
@@ -220,4 +191,4 @@ declare global {
 window.confirmDialog = confirmDialog;
 window.bootstrap = bootstrap;
```

Comment:
Here, this patch is added because we have .cshtml inline scripts. Once all these confirmModals and Bootstrap components will be abstracted we will be able to remove this. And also this will allow starting using different components other than Bootstrap ones.